examples/image.rs

```diff
@@ -1,12 +1,15 @@
+use std::collections::HashMap;
+
 use cushy::animation::ZeroToOne;
 use cushy::figures::Size;
-use cushy::value::{Dynamic, MapEachCloned, Source};
+use cushy::value::{Dynamic, MapEachCloned, Source, Switchable};
 use cushy::widget::MakeWidget;
 use cushy::widgets::image::{Aspect, ImageScaling};
 use cushy::widgets::slider::Slidable;
 use cushy::widgets::Image;
 use cushy::Run;
 use kludgine::include_texture;
+use kludgine::wgpu::FilterMode;
```

Comment:
I'll take care of this post-merge since the line above it shows I failed my own best practices: examples should use `cushy::kludgine` paths so that users can copy/paste an example into their own project and it "just work". Since I'm hopeful you might make more contributions I wanted to just mention this -- but clearly I forget to do this sometimes :)